website/settings/celeryconfig.py

```diff
@@ -0,0 +1,35 @@
+from celery.schedules import crontab
+
+
+##### Celery #####
+## Default RabbitMQ broker
+BROKER_URL = 'amqp://'
+
+# Default RabbitMQ backend
+CELERY_RESULT_BACKEND = 'amqp://'
+
+# CELERYBEAT_SCHEDULE_FILENAME = 'celeryconfig.py'
+# Setting up a scheduler, essentially replaces an
+#  independent cronjob
+CELERYBEAT_SCHEDULE = {
+    '5-minute-emails': {
+        'task': 'notify.send_users_email',
+        'schedule': crontab(minute='*/5'),
```

Comment:
@sloria, this time was from a few discussions with @erinmayhood and @jlcohoon. Some of the original thoughts on this were between 5 minutes and an hour. If you think this is worth another discussion then please set up a time with community.